src/test/java/seedu/address/ui/ResultDisplayTest.java

```diff
@@ -11,7 +11,11 @@
 
```

Comment:
Don't see any other tests written other than this. you coverage will drop over time resulting in regression errors in the later part of your project.